src/backend/distributed/deparser/deparse_statistics_stmts.c

```diff
@@ -236,9 +236,20 @@ static void
 AppendColumnNames(StringInfo buf, CreateStatsStmt *stmt)
 {
 	ColumnRef *column = NULL;
+
 	foreach_ptr(column, stmt->exprs)
 	{
-		Assert(IsA(column, ColumnRef));
+		if (!IsA(column, ColumnRef))
+		{
+			if (stmt->if_not_exists)
```

Comment:
Why do we have this? We already returned if the statistics already exist, and if the statistics do not exist we should throw the error.